### PR TITLE
Fix the format of `title` tag

### PR DESCRIPTION
**Pull request by mfuji09**
### Description

Fix the format of `title` tag, with the recent style.

### Motivation

These templates shows `title` as the old format.

### CODE REVIEW

File: files/en-us/mdn/writing_guidelines/page_structures/page_types/api_property_subpage_template/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ browser-compat: path.to.feature.NameOfTheProperty
 >
 > ```md
 > ---
-> title: NameOfTheParentInterface.NameOfTheProperty
+> title: NameOfTheParentInterface: NameOfTheProperty property
```
Comment (skyclouds2001) [inline files/en-us/mdn/writing_guidelines/page_structures/page_types/api_property_subpage_template/index.md:21]: the same `title` key of method page should also be updated (at the same position)

Comment (skyclouds2001) [inline files/en-us/mdn/writing_guidelines/page_structures/page_types/api_property_subpage_template/index.md:21]:
```suggestion
> title: "NameOfTheParentInterface: NameOfTheProperty property"
```

like the event page template, the `"` is required

the method page template is the same

File: files/en-us/mdn/writing_guidelines/page_structures/page_types/api_property_subpage_template/index.md [hunk 2]
```diff
@@ -31,8 +31,8 @@ browser-compat: path.to.feature.NameOfTheProperty
 >
 > - **title**
 >   - : Title heading displayed at the top of the page.
->     Format as _NameOfTheParentInterface_**.**_NameOfTheProperty_.
->     For example, the [`capabilities`](/en-US/docs/Web/API/VRDisplay/capabilities) property of the [`VRDisplay`](/en-US/docs/Web/API/VRDisplay) interface has a `title` of `VRDisplay.capabilities`.
+>     Format as `NameOfTheParentInterface: NameOfTheProperty property`.
```
Comment (skyclouds2001) [inline files/en-us/mdn/writing_guidelines/page_structures/page_types/api_property_subpage_template/index.md:34]: make it is better to follow the format in event page template, like:

"_NameOfTheParentInterface_**:** _NameOfTheEvent_ **event**"

```suggestion
>     Format as "NameOfTheParentInterface: NameOfTheProperty property".
```

File: files/en-us/mdn/writing_guidelines/page_structures/page_types/api_method_subpage_template/index.md [hunk 3]
```diff
@@ -31,8 +31,8 @@ browser-compat: path.to.feature.NameOfTheMethod
 >
 > - **title**
 >   - : Title heading displayed at the top of the page.
->     Format as _NameOfTheParentInterface_**.**_NameOfTheMethod_**()**.
->     For example, the [count()](/en-US/docs/Web/API/IDBIndex/count) method of the [IDBIndex](/en-US/docs/Web/API/IDBIndex) interface has a _title_ of `IDBIndex.count()`.
+>     Format as `NameOfTheParentInterface: NameOfTheMethod() method`.
```
Comment (skyclouds2001) [inline files/en-us/mdn/writing_guidelines/page_structures/page_types/api_method_subpage_template/index.md:34]:
```suggestion
>     Format as "NameOfTheParentInterface: NameOfTheMethod() method".
```

see https://github.com/mdn/content/pull/32897/files#r1545464263

Comment (hamishwillee) [reply files/en-us/mdn/writing_guidelines/page_structures/page_types/api_method_subpage_template/index.md:34]: 1. The `"` are important so this is a good catch, but the code style is still needed (in all cases) - you can see this when you look at the rendering.

```suggestion
>     Format as `"NameOfTheParentInterface: NameOfTheMethod() method"`.
```

2. I like the removal of the example here "For example, the ...". There is no real potential for ambiguity, so no real value in cross linking.

File: files/en-us/mdn/writing_guidelines/page_structures/page_types/api_method_subpage_template/index.md [hunk 4]
```diff
@@ -31,8 +31,8 @@ browser-compat: path.to.feature.NameOfTheMethod
 >
 > - **title**
 >   - : Title heading displayed at the top of the page.
->     Format as _NameOfTheParentInterface_**.**_NameOfTheMethod_**()**.
->     For example, the [count()](/en-US/docs/Web/API/IDBIndex/count) method of the [IDBIndex](/en-US/docs/Web/API/IDBIndex) interface has a _title_ of `IDBIndex.count()`.
+>     Format as `"NameOfTheParentInterface: NameOfTheMethod() method"`.
+>     For example, the [count()](/en-US/docs/Web/API/IDBIndex/count) method of the [IDBIndex](/en-US/docs/Web/API/IDBIndex) interface has a _title_ of `IDBIndex: count() method`.
```
Comment (hamishwillee) [inline files/en-us/mdn/writing_guidelines/page_structures/page_types/api_method_subpage_template/index.md:35]: Thanks very much @mfuji09 (and @skyclouds2001). This works for me - though as I mentioned, FWIW I was OK with the deletion of the example cases.

**Review by hamishwillee — APPROVED** [review]

**Pull request merged**